assets/wizards/readerRevenue/views/configure-landing-page/index.js

```diff
@@ -25,33 +23,15 @@ class ConfigureLandingPage extends Component {
 	 */
 	render() {
 		const { donationPage } = this.props;
-		const publishedIcon = (
-			<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-				<Path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z" />
-			</SVG>
-		);
-		const notPublishedIcon = (
-			<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-				<Path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-2h2v2zm0-4h-2V7h2v6z" />
-			</SVG>
-		);
 		return (
 			<Fragment>
 				{ donationPage && (
 					<Fragment>
 						{ 'publish' !== donationPage.status && (
-							<div className="newspack-memberships-page-wizard-wizard__notice">
-								{ notPublishedIcon }
-								{ __(
-									'Your donations landing page has been created, but is not yet published. You can now edit it and publish when you\'re ready.'
-								) }
-							</div>
+							<Notice isError noticeText={ __( 'Your donations landing page has been created, but is not yet published. You can now edit it and publish when you\'re ready.' ) } />
```

Comment:
There's inconsistency between Membership and Donation in these notices and buttons. It's unrelated to this PR, so probably should be resolved in a separate oone.

Reply:
I agree, it should be resolved in a separate PR.